app-modules/service-management/tests/ServiceRequest/CreateServiceRequestTest.php

```diff
@@ -239,3 +273,167 @@
 
     expect($serviceRequest->division->id)->toEqual($request['division_id']);
 });
+
+test('cannot create service requests if user is not an auditor or manager of any service request type', function () {
+    $settings = app(LicenseSettings::class);
+
+    $settings->data->addons->serviceManagement = false;
+
+    $settings->save();
+
+    $user = User::factory()->licensed(LicenseType::cases())->create();
+
+    actingAs($user)
+        ->get(
+            ServiceRequestResource::getUrl('create')
+        )->assertForbidden();
+
+    $user->givePermissionTo('service_request.view-any');
+    $user->givePermissionTo('service_request.create');
+
+    $team = Team::factory()->create();
+
+    $user->teams()->attach($team);
+
+    $user->refresh();
+
+    $settings->data->addons->serviceManagement = true;
+
+    $settings->save();
+
+    livewire(CreateServiceRequest::class)
+        ->assertForbidden();
+
+    actingAs($user)
+        ->get(
+            ServiceRequestResource::getUrl('create')
+        )->assertForbidden();
+});
+
+test('displays only service request types managed by the current user', function () {
+    $settings = app(LicenseSettings::class);
+
+    $settings->data->addons->serviceManagement = true;
+
+    $settings->save();
+
+    $user = User::factory()->licensed(LicenseType::cases())->create();
+
+    $team = Team::factory()->create();
+
+    $user->teams()->attach($team);
+
+    $user->refresh();
+
+    $user->givePermissionTo('service_request.view-any');
+    $user->givePermissionTo('service_request.create');
+
+    actingAs($user);
+
+    $serviceRequestTypesWithManagers = ServiceRequestType::factory()
+        ->hasAttached($team, [], 'managers')
+        ->create();
+
+    $serviceRequestTypesWithoutManagers = ServiceRequestType::factory()->create();
+
+    livewire(CreateServiceRequest::class)
+        ->assertFormFieldExists('type_id', function (Select $field) use ($serviceRequestTypesWithManagers, $serviceRequestTypesWithoutManagers): bool {
+            $options = $field->getOptions();
+
+            return in_array($serviceRequestTypesWithManagers->getKey(), array_keys($options)) &&
+                   ! in_array($serviceRequestTypesWithoutManagers->getKey(), array_keys($options));
+        });
+});
+
+test('create service requests if user is an auditor or manager of any service request type', function () {
```

Comment:
The name and/or content should be changed to only focus on managers. Only managers can create